__tests__/app.test.js

```diff
@@ -36,4 +36,36 @@ describe('GET /api', () => {
             expect(res.body).toEqual(endpoints)
         })
     })
+})
+
+describe('GET /api/reviews/:review_id', () => {
+    test('GET - STATUS: 200 - responds with review object of corresponding review_id endpoint', () => {
```

Comment:
to allow for later extension of the endpoint, this test can use a flexible matcher eg expect.objectContaining or .toMatchObject